shared/oae/js/activityadapter.js

```diff
@@ -40,11 +40,13 @@ var _expose = function(exports) {
      * @param  {Function}               sanitization.encodeForHTML              Encode a value such that it is safe to be embedded into an HTML tag
      * @param  {Function}               sanitization.encodeForHTMLAttribute     Encode a value such that it is safe to be embedded into an HTML attribute
      * @param  {Function}               sanitization.encodeForURL               Encode a value such that it is safe to be used as a URL fragment
+     * @param  {Object}                 [opts]                                  Optional arguments
+     * @param  {String}                 [opts.resourceHrefOverride]             When specified, this value will replace any URL specified for an entity. This can be used to control outbound links in different contexts (e.g., email invitation)
```

Comment:
An example is difficult, but I'll make it more clear that, for any `<a...` tag in the activity html, it will replace the `href` attribute with whatever is specified in that option.